lib/controllers/meeting.js

```diff
@@ -0,0 +1,23 @@
+const Meeting = require("../models/meeting");
+
+module.exports = {
+  display: (req, res) => {
+    try {
+      const meeting = await Meeting.findOne({ "_id": id });
+      res.sendStatus(200);
```

Comment:
Almost there, just need to grab the `id` from the request body to make the query with. Then send back the meeting which *should* be json with `res.status( 200 ).send( meeting )`.
```suggestion
  display: (req, res) => {
    try {
      const meeting = await Meeting.findById( req.body.id );
      res.status( 200 ).json( meeting );
```